app/uk/gov/hmrc/agentsubscriptionfrontend/views/confirm_your_agency.scala.html

```diff
@@ -4,29 +4,28 @@
 @(registrationName: String, postcode: String, utr: Utr, nextPageUrl: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @uk.gov.hmrc.agentsubscriptionfrontend.views.html.main_template(appConfig, title = Messages("confirmYourAgency.title"), bodyClasses = None) {
-  <h1>@Messages("confirmYourAgency.title")</h1>
+    <header class="page-header">
+        <h1>@Messages("confirmYourAgency.title")</h1>
+    </header>
 
-  <table class="form-group">
-    <thead>
-    <tr>
-      <th colspan="2"><span class="visuallyhidden">Agency details</span></th>
-    </tr>
-    </thead>
-    <tbody>
-    <tr>
-      <td>Name</td>
-      <td id="registration-name">@registrationName</td>
-    </tr>
-    <tr>
-      <td>Postcode</td>
-      <td id="postcode">@postcode</td>
-    </tr>
-    <tr>
-      <td>UTR</td>
-      <td id="utr">@{utr.value}</td>
-    </tr>
-    </tbody>
-  </table>
-  <a id="continue" class="button" href="@nextPageUrl">Yes</a>
-  <a id="return" class="button--secondary" href="@CheckAgencyController.showCheckAgencyStatus">No</a>
+    <table class="form-group">
+        <caption class="visuallyhidden">@Messages("confirmYourAgency.table.caption")</caption>
+        <tbody>
+            <tr>
+                <th>@Messages("confirmYourAgency.table.th.name")</th>
```

Comment:
Perhaps it would be better to make the message key describe what the message is for rather than where it is in the markup? That way it wouldn't need to change if the markup changed (e.g. to not use a table).

`confirmYourAgency.agency.name` might seem like a reasonable name but it's actually a bad name because these details are NOT agency details, they are registered business details. The agency details are only known to HMRC's systems when we add them later in this this subscription process.


`confirmYourAgency.name` seems like the best name to me, or maybe `confirmYourAgency.registration.name` or `confirmYourAgency.business.name`?